src/lib/charts/ChartLegend/ChartLegend.tsx

```diff
@@ -1,25 +1,31 @@
-import React, { FC, HTMLAttributes } from 'react';
```

Comment:
FYI, I moved the old `ChartLegend` code to `ChartLegendWrapper` - I didn't get rid of it.